int32 min_observations_per_point
---
# No response